Source/monstdat.cpp

```diff
@@ -56,7 +56,7 @@ const MonsterData MonstersData[] = {
 /* MT_RSKELSD */ { P_("monster", "Burning Dead Captain"),    "SkelSd\\SklSr",    nullptr,          nullptr,             MonsterAvailability::Always,   128,    575, true,        false,       { 13,  8, 12,  7, 15, 16 }, { 4, 1, 1, 1, 1, 1 },         3,        5,      6,     16,     30, AI_SKELSD,    0                                                                  ,    2,   35,      8,          4,         10,     0,       0,           0,           0,          15, MonsterClass::Undead,   IMMUNE_MAGIC | RESIST_FIRE |                    IMMUNE_NULL_40  , IMMUNE_MAGIC | IMMUNE_FIRE |                    IMMUNE_NULL_40    ,  3,                   0,          393 },
 /* MT_XSKELSD */ { P_("monster", "Horror Captain"),          "SkelSd\\SklSr",    nullptr,          "SkelSd\\Black",     MonsterAvailability::Always,   128,    575, true,        false,       { 13,  8, 12,  7, 15, 16 }, { 4, 1, 1, 1, 1, 1 },         4,        6,      8,     35,     50, AI_SKELSD,                                     MFLAG_SEARCH                      ,    3,   40,      8,          5,         14,     0,       0,           0,           0,          30, MonsterClass::Undead,   IMMUNE_MAGIC |               RESIST_LIGHTNING | IMMUNE_NULL_40  , IMMUNE_MAGIC |               RESIST_LIGHTNING | IMMUNE_NULL_40    ,  3,                   0,          604 },
-/* MT_INVILORD*/ { P_("monster", "Invisible Lord"),          "TSneak\\TSneak",   "TSneak\\Sneakl", nullptr,             MonsterAvailability::Never,    128,    800, false,       false,       { 13, 13, 15, 11, 16,  0 }, { 2, 1, 1, 1, 1, 1 },        19,       20,     14,    278,    278, AI_SKELSD,                                     MFLAG_SEARCH | MFLAG_CAN_OPEN_DOOR,    3,   65,      8,         16,         30,     0,       0,           0,           0,          60, MonsterClass::Demon,    RESIST_MAGIC | RESIST_FIRE | RESIST_LIGHTNING | IMMUNE_NULL_40  , RESIST_MAGIC | RESIST_FIRE | RESIST_LIGHTNING | IMMUNE_NULL_40    ,  3,                   0,         2000 },
+/* MT_INVILORD*/ { P_("monster", "Invisible Lord"),          "TSneak\\TSneak",   nullptr,          nullptr,             MonsterAvailability::Never,    128,    800, false,       false,       { 13, 13, 15, 11, 16,  0 }, { 2, 1, 1, 1, 1, 1 },        19,       20,     14,    278,    278, AI_SKELSD,                                     MFLAG_SEARCH | MFLAG_CAN_OPEN_DOOR,    3,   65,      8,         16,         30,     0,       0,           0,           0,          60, MonsterClass::Demon,    RESIST_MAGIC | RESIST_FIRE | RESIST_LIGHTNING | IMMUNE_NULL_40  , RESIST_MAGIC | RESIST_FIRE | RESIST_LIGHTNING | IMMUNE_NULL_40    ,  3,                   0,         2000 },

```

Comment:
I take it "TSneak\\Sneakl" does not exist in the MPQ, right?

Reply:
Shouldn't we either keep stuff like this intact for potential modders, or remove it altogether? I don't see any reason why some fields should go, but others should not.

Reply:
We should remove it. Modders can always fish out the data and partial routines and reimplement them.

The reasoning here is that I knew these to have no usage and no consequence for removing them, where I'm less sure about the rest. But having this removed should make the rest easier to reason about.